idaes/power_generation/unit_models/helm/turbine_outlet.py

```diff
@@ -60,16 +60,29 @@ def build(self):
         self.flow_coeff.fix()
         self.efficiency_mech.fix()
 
+
+        self.tel_c0 = Var(initialize=0.0064)
```

Comment:
Minor nitpicks, but a doc string and explicit units (`units=pyunits.dimensionless`) would be nice.

Reply:
Thanks.  I was kind of lazy with the units there.  I moved the units to the coefficients where they belong, and added doc strings.  @andrewlee94 and @MAZamarripa should be okay to re-review now.